TuviSRPLib/TuviSRPLib/ProtonSRPClient.cs

```diff
@@ -48,6 +48,34 @@ public ProtonSRPClient()
 	     */
         public virtual void Init(BigInteger N, BigInteger g, IDigest digest, SecureRandom random)
         {
+            byte[] bytesBorder = new byte[] {
```

Comment:
'System.Linq.Enumerable.Repeat' can generate a sequence that contains one repeated value.
Maybe it's more readable
```
int bitSize = 2048;
int bitInByte = sizeof(byte) * 8;
int count = bitSize / bitInByte;
        
byte[] bytes = Enumerable.Repeat(byte.MaxValue, count).ToArray();
bytes[0] = 0x7f;
```

Reply:
I agree with Alex